iSH.xcodeproj/project.pbxproj

```diff
@@ -1199,11 +1199,9 @@
 				TargetAttributes = {
 					497F6CE3254E5E4C00C82F46 = {
 						CreatedOnToolsVersion = 12.2;
-						DevelopmentTeam = CK5SXRTBR7;
```

Comment:
Intentional changes?

Reply:
In a way but it should be removed since the dev team stayed the same